easybuild/tools/filetools.py

```diff
@@ -1242,7 +1242,7 @@ def calc_block_checksum(path, algorithm):
     return algorithm.hexdigest()
 
 
-def verify_checksum(path, checksums):
+def verify_checksum(path, checksums, computed_checksums=None):
```

Comment:
@Flamefire Please document this additional optional argument in the docstring below

Reply:
Done. Also updated the `checksums` docstring to make the difference clearer